readme.md

```diff
@@ -203,9 +203,36 @@ Parse response body with `JSON.parse` and set `accept` header to `application/js
 
 ###### query
 
-Type: `string` `Object` [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)
+Type: `string` `Object<string, string>` [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)
 
-Query string object that will be added to the request URL. This will override the query string in `url`.
+Query string that will be added to the request URL. This will override the query string in `url`.
+
+If you need to pass in an array, you can do it with a `URLSearchParams` instance:
+
+```js
+const got = require('got');
+
+const query = new URLSearchParams([['key', 'a'], ['key', 'b']]);
+
+got('https://example.com', {query});
```

Comment:
you don't have to create a new `URLSearchParams` instance... it'll be done in `normalize-arguments.js`.

```js
got('https://example.com', {query: [['key', 'a'], ['key', 'b']]});
```

Reply:
That's exactly what I want to remove in the next major version. Then it will be; either use an object with string values or specify a `URLSearchParams` instance.

Reply:
So either `query: {foo: '1'}` or `new URLSearchParams({foo: 1})`

Reply:
okay :D